config.js.example

```diff
@@ -0,0 +1,11 @@
+"use strict";
```

Comment:
Shouldn’t this file be TypeScript?

Reply:
So, this file isn't even used anymore so will be deleted, but to answer the question, no it shouldn't be typescript. When I converted from compiled in repo to user compilation, I saw that the `setup.ts` file would not be compiled to use before `config.ts` is compiled. So what it did before (and kinda similarly now) is that it would fill in the file from `config.js.example` and write it to the `dist/config/config.js` file. Now it gets the compiled `config.js` replaces it's content with correct details and gets `config.ts` and also replaces contents with correct details (basically replace the details in compiled version to not compile again). So on the next compile (or update) it will automatically compile with correct content.

Reply:
It seems like a complicated way to do this, is there any other way that is less complicated? :slightly_smiling_face: